index.md

```diff
@@ -106,84 +106,104 @@ Window Azure
+No work day is complete without **pushing or merging a pull request.**  
+When I have free time, I spend it on **_free/libre_ software [gnu.org](https://www.gnu.org/)**
 
 - ![Ton](./assets/images/Ton.png){: .animated}  
 #### Ton  
 ##### Senior Developer
-No work day is complete without **Scrum meeting.**  
-When I have free time I spend time on **push-up and sit-up**
+No work day is complete without **a Scrum meeting.**  
+When I have free time I spend it doing **push-ups and sit-ups.**
 
 - ![Orc](./assets/images/Orc.png){: .animated}  
 #### Orc  
 ##### Senior Developer
 No work day is complete without **TASKS**  
-When I have free time I spend time on **PLAYING A GAME**
+When I have free time I spend it **PLAYING GAMES**
 
 - ![Note](./assets/images/Note.png){: .animated}
 #### Note   
 ##### Senior Developer
-No work day is complete without **some nice music**  
-When I have free time I spend time on **finding new songs**
+No work day is complete without **some nice music.**  
+When I have free time I spend it **finding new songs.**
 
 - ![Phyo](./assets/images/Phyo.png){: .animated}  
 #### Phyo
 ##### Software Developer
-No work day is complete without **a cup of coffee**  
-When I have free time I spend time on **reading or watching football match.**
+No work day is complete without **a cup of coffee.**  
+When I have free time I spend it **reading or watching football matches.**
 
 
 {: .photo02 .animated}
 - ![Koon](./assets/images/Koon.png){: .animated}
 #### Koon    
 ##### Software Developer
 No work day is complete without **some great food.**  
-When I have free time I spend time on **workout and traveling.**
+When I have free time I spend it **working out and traveling.**
 
 - ![Santi](./assets/images/Santi.png){: .animated}
 #### Santi    
 ##### Graphic designer
-No work day is complete without **amusement**  
-When I have free time I spend time on **listening to music**
+No work day is complete without **amusement.**  
+When I have free time I spend it **listening to music.**
 
 - ![Fai](./assets/images/Fai.png){: .animated}
 #### Fai    
 ##### Software Developer
-No work day is complete without **learning**  
-When I have free time I spend time on **plant trees, read books and watch movies**
+No work day is complete without **learning.**  
+When I have free time I spend it **planting trees, reading books, and watching movies.**
 
 - ![Pek](./assets/images/Pek.png){: .animated}
 #### Pek    
 ##### Software Developer
-No work day is complete without **some snack**  
-When I have free time I spend time on **reading and watching youtube channel**
+No work day is complete without **some snacks.**  
+When I have free time I spend it **reading and watching YouTube channels.**
+
+- ![James](./assets/images/James.png){: .animated}
+#### James    
+##### Software Developer
+No work day is complete without **chocolate bar.**
+When I have free time I spend **time on practice programming skill**
```

Comment:
No work day is complete without **a bar of chocolate**
When I have free time I spend time on **practicing programming skill.**